src/timetable/timetable-row/row-bar.tsx

```diff
@@ -125,7 +120,7 @@ export const RowBar = ({
     ]);
 
     return (
-        <RowCellContainer data-testid={testId} {...otherProps}>
+        <RowCellContainer data-testid="timetable-row" {...otherProps}>
```

Comment:
getting this error:
```Warning: Unknown event handler property `onRowNameClick`. It will be ignored.```

I believe it originates from here, we'll need to exclude passing the custom props to the component

Reply:
noted, have removed `otherProps` entirely as it was also unintentionally passing in things like `name` and `subtitle`